src/Symfony/Component/Messenger/Tests/DependencyInjection/MessengerPassTest.php

```diff
@@ -0,0 +1,172 @@
+            ->addTag('messenger.receiver')
+        ;
+
+        (new MessengerPass())->process($container);
+
+        $handlerLocatorDefinition = $container->getDefinition($container->getDefinition('messenger.handler_resolver')->getArgument(0));
+        $this->assertSame(ServiceLocator::class, $handlerLocatorDefinition->getClass());
+        $this->assertEquals(
+            array('handler.'.DummyMessage::class => new ServiceClosureArgument(new Reference(DummyHandler::class))),
+            $handlerLocatorDefinition->getArgument(0)
+        );
+
+        $this->assertEquals(
+            array(DummyReceiver::class => new Reference(DummyReceiver::class)),
+            $container->getDefinition('messenger.receiver_locator')->getArgument(0)
+        );
+    }
+
+    /**
+     * @expectedException \Symfony\Component\DependencyInjection\Exception\RuntimeException
+     * @expectedExceptionMessage Invalid handler service "Symfony\Component\Messenger\Tests\DependencyInjection\UndefinedMessageHandler": message class "Symfony\Component\Messenger\Tests\DependencyInjection\UndefinedMessage" used as argument type in method "Symfony\Component\Messenger\Tests\DependencyInjection\UndefinedMessageHandler::__invoke()" does not exist.
+     */
+    public function testUndefinedMessageClassForHandler()
+    {
+        $container = $this->getContainerBuilder();
+        $container
+            ->register(UndefinedMessageHandler::class, UndefinedMessageHandler::class)
+            ->addTag('message_handler')
+        ;
+
+        (new MessengerPass())->process($container);
+    }
+
+    /**
+     * @expectedException \Symfony\Component\DependencyInjection\Exception\RuntimeException
+     * @expectedExceptionMessage Invalid handler service "Symfony\Component\Messenger\Tests\DependencyInjection\NotInvokableHandler": class "Symfony\Component\Messenger\Tests\DependencyInjection\NotInvokableHandler" must have an "__invoke()" method.
+     */
+    public function testNotInvokableHandler()
+    {
+        $container = $this->getContainerBuilder();
+        $container
+            ->register(NotInvokableHandler::class, NotInvokableHandler::class)
+            ->addTag('message_handler')
+        ;
+
+        (new MessengerPass())->process($container);
+    }
+
+    /**
+     * @expectedException \Symfony\Component\DependencyInjection\Exception\RuntimeException
+     * @expectedExceptionMessage Invalid handler service "Symfony\Component\Messenger\Tests\DependencyInjection\MissingArgumentHandler": method "Symfony\Component\Messenger\Tests\DependencyInjection\MissingArgumentHandler::__invoke()" must have exactly one argument corresponding to the message it handles.
+     */
+    public function testMissingArgumentHandler()
+    {
+        $container = $this->getContainerBuilder();
+        $container
+            ->register(MissingArgumentHandler::class, MissingArgumentHandler::class)
+            ->addTag('message_handler')
```

Comment:
This tag has changed and is now `messenger.message_handler`. Could you rebase this PR on master and update the tests?